_sass/_fonts.scss

```diff
@@ -75,6 +75,10 @@
   padding-right: $base-padding-lite;
 }
 
+.icon-padded_after:before {
```

Comment:
I think Hound is trying to make a distinction between `pseudo elements` and `pseudo classes`. I have the distinction, but until now, I never took the time to investigate. This is what I discovered(!):

From https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements

> Sometimes you will see double colons (::) instead of just one (:). This is part of CSS3 and an attempt to distinguish between pseudo-classes and pseudo-elements. Most browsers support both values.

`pseudo elements` like before and after are more/less elements
 `pseudo classes` are essentially metadata applied to an element `:first-of-type`